utils/sim/mos-sim.c

```diff
@@ -185,11 +195,24 @@ int main(int argc, const char *argv[]) {
     }
   }
 
+  char status_buf[9];
+  status_buf[8] = '\0';
+
+  const char statuses[] = "czidb1vn";
```

Comment:
nit: Move these both into the loop right before they're used. Moving them out is the compiler's job :P